Code/Framework/AzFramework/Platform/Linux/platform_nativeui_linux.cmake

```diff
@@ -27,6 +27,7 @@ if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
             3rdParty::X11::xkbcommon
             3rdParty::X11::xkbcommon_X11
             xcb-xinput
+            xcb-keysyms
```

Comment:
does this mean that compilers are going to need to get more packages from their distro?

Reply:
well, we need the xcb-keysyms package.

Reply:
maybe we would load the library dynamically and use that function, but the headers will be needed to be able to compile